app/components/support_interface/reason_for_rejection_dashboard_section_component.html.erb

```diff
@@ -0,0 +1,27 @@
+<h3 class="govuk-heading-m govuk-!-font-size-27"><%=@heading%></h3>
+<div class="govuk-grid-row govuk-!-margin-bottom-8">
+  <div class="govuk-grid-column-one-half">
+    <%= render SupportInterface::TileComponent.new(
+      count: @percentage_rejected,
+      label: 'of all rejections',
```

Comment:
'of all structured rejections'?

Reply:
I'm possibly overthinking this, but I think your suggestion is a little bit clearer, so let's go with that. Then we can :shipit: